spec/garage_spec.rb

```diff
@@ -1,12 +1,27 @@
-require_relative '../lib/garage'
+require_relative "../lib/garage"
+require_relative "../lib/bike_container"
 
 describe Garage do
 
-let(:bike) {Bike.new}
-let(:garage) {Garage.new}
+  let(:garage) {Garage.new}
+  let(:bike) {Bike.new}
+  let(:van) {Van.new}
 
-	it "should fix bikes instantaneously once received" do
-	garage.dock(bike)
-	garage.fix_bikes		
-	end 
-end 
\ No newline at end of file
+  it "should accept a bike and fix it" do
```

Comment:
I've now broken any bike that comes to the garage and once it has been docked in using 'accept' it should be fixed instantaneously